app/renderer/unreads.js

```diff
@@ -5,9 +5,9 @@ const { ipcRenderer: ipc } = require('electron');
 
 const seenMessages = new Map();
 
-function keyByMessage({ subject, sender }) {
+function keyByMessage({ subject, sender, convlen }) {
```

Comment:
Let's rename `convlen` as `conversationLength`. I spent 5 seconds until I realize what it means and where it comes from.

```suggestion
function keyByMessage({ subject, sender, conversationLength }) {
```